app/tests/components_tests/test_views.py

```diff
@@ -442,3 +444,51 @@ def test_display_set_bulk_delete(
     assert base_obj.civ_sets_related_manager.count() == 3
     assert ob1 not in base_obj.civ_sets_related_manager.all()
     assert ob2 not in base_obj.civ_sets_related_manager.all()
+
+
+@pytest.mark.django_db
+def test_display_ci_example_value(client):
+
+    sc = {
+        "$id": "https://example.com/person.schema.json",
+        "$schema": "https://json-schema.org/draft/2020-12/schema",
+        "title": "Person",
+        "type": "object",
+        "properties": {
+            "firstName": {
+                "type": "string",
+                "description": "The person's first name.",
+            },
+            "lastName": {
+                "type": "string",
+                "description": "The person's last name.",
+            },
+            "age": {
+                "description": "Age in years which must be equal to or greater than zero.",
+                "type": "integer",
+                "minimum": 1,
+            },
+        },
+    }
```

Comment:
Why bother with a schema here? It is not used by your test at all.

Reply:
Done